android/pom.xml

```diff
@@ -211,7 +211,7 @@
         </plugin>
         <plugin>
           <artifactId>maven-jar-plugin</artifactId>
-          <version>3.2.0</version>
+          <version>3.3.0</version>
```

Comment:
[`3.4.0`](https://issues.apache.org/jira/browse/MJAR-292) is easier to configure for MRJARs, which this PR produces. Additionally, [`3.3.0`](https://github.com/apache/maven-jar-plugin/releases/tag/maven-jar-plugin-3.3.0) includes provenance fixes for module artifacts.

Reply:
(also, why is this in the android project? lol)